lib/features/dragging/HoverFix.js

```diff
@@ -39,12 +39,21 @@ export default function HoverFix(eventBus, dragging, elementRegistry) {
       return;
     }
 
-    var gfx = self._findTargetGfx(event);
+    var originalEvent = event.originalEvent;
+
+    var gfx = self._findTargetGfx(originalEvent);
 
     if (gfx) {
       var element = elementRegistry.get(gfx);
 
+      // 1) cancel current mousemove
+      event.stopPropagation();
+
+      // 2) emit fake hover for new target
       dragging.hover({ element: element, gfx: gfx });
+
+      // 3) re-trigger move event
+      dragging.move(originalEvent);
```

Comment:
This should also make listening for `create.hover` in `Create` to update the rules obsolete, right?

Reply:
No, we still need it for container elements.

Yesterday we found out the browser fires an `out` event for the `root` and a `hover` event for the container element when after drag mode is initiated. Therefore the `new-parent marker` would be removed from the actual hover (the container element). This was the actual reason we added the `create.hover` listening.

Reply:
Forgot that in the mean time, thanks for pointing it out.